README.md

```diff
@@ -624,6 +624,7 @@ exchange | Preview the whole HTTP exchange(request and response)
 * `rest-client.logLevel`: The verbosity of logging in the REST output panel. (Default is __error__)
 * `rest-client.enableSendRequestCodeLens`: Enable/disable sending request CodeLens in request file. (Default is __true__)
 * `rest-client.enableCustomVariableReferencesCodeLens`: Enable/disable custom variable references CodeLens in request file. (Default is __true__)
+* `rest-client.retry`: Number of retry request. (Default is __0__)

```

Comment:
It seems not clear for me that when will the retry operation occur if we set the value to larger than 0? I think the behavior should be documented

Reply:
Hi @Huachao

> The number limit of retry requests. The default is turned off with 0. 

Is it clear? Any advice for the document better. Thank

Reply:
@chuongtrh I mean as an extension user,  how can I know in which circumstance the retry operation will happen.

Reply:
@Huachao Base on the document from [got](https://github.com/sindresorhus/got#retry). When response with status **500**, the extension will be retry 2 times because the limit default is 2. I think it is good if extension has option setting retry limit.

Reply:
@chuongtrh are you sure **got**  package behaves as you described? https://github.com/sindresorhus/got/blob/master/readme.md#retry

Reply:
@Huachao yes. I make a simple request test like the image above

![image](https://user-images.githubusercontent.com/7924855/70917905-bedf5000-2050-11ea-856e-af46d60e544c.png)
![image](https://user-images.githubusercontent.com/7924855/70917993-ea623a80-2050-11ea-9031-7b80197944a6.png)

Reply:
@chuongtrh from the doc link I have posted, **500** is just one possible reason for retry happens

Reply:
@Huachao Correct. 
I found this situation when I try to write to the log . It will be writing 3 times when making a request via rest-client. It is hard when debugging. Play with postman only 1 time

This option is not must have but have is good :) Thank for the quick reply.